### DISCUSSION TITLE
Annoying false alarm Trojan detected in VirusTotal

### DISCUSSION THREAD

**Discussion by chanh2018** [LawnchairLauncher/lawnchair#3899] [Q&A] [answered]
Can this false positive be removed in next debug version? 
VirusTotal detected this, it's not real but annoying.
K7GW Trojan ( 004a8ed81 )

**Comment by MrSluffy** [comment, marked as answer]
I'm not sure how virus total works, we can't remove it since Virus Total doesnt point out which code was identify as trojan, maybe try reporting in virus total and linked the lawnchair repo. or maybe when lawnchair pay the virus total maybe they will whitelist lawnchair lol
